bin/client/main.rs

```diff
@@ -188,9 +180,16 @@ async fn get_all<C: Cache + Send + Sync + Clone>(cache: C, opts: Get) -> Result<
         .as_ref()
         .unwrap_or(&zero_vec)
         .iter()
-        .map(|p| (p.label.sha256.clone(), cache.clone(), parcels.clone()))
-        .map(|(sha, c, parcels)| async move {
-            match c.get_parcel(&sha).await {
+        .map(|p| {
```

Comment:
Ooo! Returning the tuple! That's a cool way of doing things!

Reply:
Yeah, I finally figured out how to pass these things properly into an `async move`